rules/Proj1102.md

```diff
@@ -0,0 +1,57 @@
+﻿---
+parent: Other
+ancestor: MSBuild
+---
+
+# Proj1102: Use Coverlet Collector or MSBuild
+Coverlet is a cross-platform code coverage framework for .NET, with support for
+line, branch and method coverage. The packages `coverlet.collector` and 
+`coverlet.msbuild` have the same purpose but should not be used together. Note
+that `coverlet.collector` is preferred over the `coverlet.msbuild`.
```

Comment:
Why is this preference? Personally I prefer `coverlet.msbuild`, since it allows to configure all the settings from the `.csproj`. Where as for `coverlet.collector` you have to do this from the command line (afaik).

Reply:
There are issues with the latter, hence it is what coverlet advises. But the rule will not force you to choose one.